tests/xilinx_test.sh

```diff
@@ -82,14 +82,16 @@ run_all() {
     run_sample fpga/streaming_memory streamingcomp_1 1
 
     ## BLAS
-    run_sample blas/nodes/axpy_test axpy_test_fpga_1_w4_1 1 --target fpga 
+    run_sample blas/nodes/axpy_test axpy_test_fpga_1_w4_1 1 --target fpga
     run_sample blas/nodes/dot_test dot_FPGA_PartialSums_float_w16_1 1 --target xilinx
     run_sample blas/nodes/gemv_test gemv_FPGA_TilesByColumn_float_True_w4_1 1 --target tiles_by_column --transpose --vectorize 4
-    run_sample blas/nodes/gemv_test gemv_FPGA_Accumulate_float_False_w4_1 1 --target accumulate --vectorize 4 
-    run_sample blas/nodes/ger_test ger_test_1 1 --target fpga 
+    run_sample blas/nodes/gemv_test gemv_FPGA_Accumulate_float_False_w4_1 1 --target accumulate --vectorize 4
+    run_sample blas/nodes/ger_test ger_test_1 1 --target fpga
 
-    # Multiple gearboxing
-    run_sample fpga/multiple_veclen_conversions multiple_veclen_conversions 0
```

Comment:
Should this be removed?

Reply:
Nope it shouldn't have been, my bad! Fixed in https://github.com/carljohnsen/dace/commit/2d647cb52deb924ad479f9de26198de79f38d1be